README.md

```diff
@@ -105,9 +101,15 @@ To use your own word embeddings or vocabulary set, specify the following paramet
 - **vocab_file**: set the file path to a predefined vocabulary set that contains lines of words.
 
 
-### Evaluation
-In the evaluation progress, you can evaluate the model with a set of evaluation metrics. Set `monitor_metrics` to define what you want to print on the screen. `val_metric` is the metric for picking the best model. Example:
+### Validation
+In the validation progress, you can evaluate the model with a set of evaluation metrics. Set `monitor_metrics` to define what you want to print on the screen. `val_metric` is the metric for picking the best model. Example:
 ```yaml
 monitor_metrics: [P@1, P@3, P@5]
 val_metric: P@1
 ```
+
+### Evaluation
+In the evaluation progress, you can evaluate a model from a pre-obtained checkpoint.
+```
+python main.py --eval --load_checkpoint CHECKPOINT_PATH --test_path TEST_DATA_PATH
```

Comment:
```suggestion
python main.py --config CONFIG_PATH --eval --load_checkpoint CHECKPOINT_PATH --test_path TEST_DATA_PATH
```

Reply:
We don't need a config file in evaluation mode now.

Reply:
Or should we add parameters like `--filter_size` to avoid size mismatch when loading `state_dict`?

Reply:
Nice catch. Specifying a config file would be easier for now.